server/common/Utils.h

```diff
@@ -258,6 +258,16 @@ class Utils {
 	 */
 	static void conv(uint64_t &dest, const std::string &src);
 
+	/**
+	 * Calculate addition with the number as a string.
+	 *
+	 * @param num0 A number represented by a string.
+	 * @param num1 An another number.
+	 *
+	 * @return A calculated result.
+	 */
+	static uint64_t add(const std::string &num0, const uint64_t num1);
```

Comment:
I take the ashie-san's suggestion.
In the context this method is used, sum() seems to be natural.
I think 'add()' is typically used as a non-static method such as
<pre>
X obj = x;
obj.add(5);
</pre>